scripts/renovate.sh

```diff
@@ -18,6 +18,24 @@ curl -o "$HOME/go.tar" -L "https://golang.org/dl/go$GO_VERSION.linux-amd64.tar.g
 tar -C "$HOME" -xzf "$HOME/go.tar"
 echo "Go $GO_VERSION has been successfully installed."
 
+# Update appVersion in graph-network-indexer Chart.yaml
```

Comment:
Could you hold-off on the renovate part for a bit?
Don´t want to block this PR, but also can't get behind this approach until I'm convinced there is no other way (that may be the case).